app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
ok, so the alerts server side does require the header being set:

```bash
$curl -s -u jdoe:password 'http://localhost:8080/hawkular/alerts'
{"errorMsg":"The HTTP header Hawkular-Tenant has to be provided."}
```

So now I don't understand how this could have worked before.